cmd/operator-sdk/new/cmd.go

```diff
@@ -308,8 +309,13 @@ func doHelmScaffold() error {
 		return fmt.Errorf("failed to create helm chart: %s", err)
 	}
 
-	valuesPath := filepath.Join("<project_dir>", helm.HelmChartsDir, chart.GetMetadata().GetName(), "values.yaml")
-	crSpec := fmt.Sprintf("# Default values copied from %s\n\n%s", valuesPath, chart.GetValues().GetRaw())
+	valuesPath := filepath.Join("<project_dir>", helm.HelmChartsDir, chart.Name(), "values.yaml")
+
+	rawValues, err := yaml.Marshal(chart.Values)
+	if err != nil {
+		return fmt.Errorf("failed to get raw chart values: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to get raw chart values: %v", err)
```
Not strictly necessary but at least it gives us the option of unwrapping in the caller, and aligns with other `fmt.Errorf`'s in the codebase.

Reply:
To do unwrapping, I'm pretty sure we need `%w` (right?), but yeah I agree we should just move all `fmt.Errorf()` and `errors.Wrapf()` calls to this. I'll update this one and any others I stumble across in the Helm code.

Reply:
I also created #2308 to track the follow-on work.

Reply:
If `err`'s underlying type implements `Unwrap`, `%v` will still allow unwrapping. Agreed though, all errors should be wrapped where it makes sense.